osdsctl/cli/volume.go

```diff
@@ -109,13 +110,14 @@ func init() {
 	volumeListCommand.Flags().StringVarP(&volProfileId, "profileId", "", "", "list volume by profile id")
 	volumeListCommand.Flags().StringVarP(&volGroupId, "groupId", "", "", "list volume by volume group id")
 
-	volumeCommand.PersistentFlags().StringVarP(&profileId, "profile", "p", "", "the id of profile configured by admin")
+	volumeCommand.PersistentFlags().StringVarP(&profileId, "profile", "", "", "the id of profile configured by admin")
```

Comment:
Please see if we can maintain previous option -p, and use different option for "--pool/-p"

Reply:
done